DigitalLearningSolutions.Web/Views/SuperAdmin/Users/UserCentreAccounts.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.UserCentreAccounts
+@using DigitalLearningSolutions.Web.ViewComponents
+@model UserCentreAccountRoleViewModel
+@{
+    ViewData["Title"] = "UserCentreAccounts";
+}
+
+@*@section NavBreadcrumbs {
+  <partial name="~/Views/" />
```

Comment:
This can be implemented without a partial. We just need a breadcrumb link with the label "User accounts" and `asp-controller="Users" asp-action="Index"`

Reply:
Review comments implemented and checked in. please review.